django-stubs/contrib/admin/options.pyi

```diff
@@ -107,18 +107,18 @@ class BaseModelAdmin(Generic[_ModelT]):
     def formfield_for_manytomany(
         self, db_field: ManyToManyField, request: HttpRequest, **kwargs: Any
     ) -> ModelMultipleChoiceField | None: ...
-    def get_autocomplete_fields(self, request: HttpRequest) -> Sequence[str]: ...
+    def get_autocomplete_fields(self, request: HttpRequest) -> _ListOrTuple[str]: ...
     def get_view_on_site_url(self, obj: _ModelT | None = ...) -> str | None: ...
     def get_empty_value_display(self) -> SafeString: ...
-    def get_exclude(self, request: HttpRequest, obj: _ModelT | None = ...) -> Sequence[str] | None: ...
+    def get_exclude(self, request: HttpRequest, obj: _ModelT | None = ...) -> _ListOrTuple[str] | None: ...
     def get_fields(self, request: HttpRequest, obj: _ModelT | None = ...) -> _FieldGroups: ...
     def get_fieldsets(self, request: HttpRequest, obj: _ModelT | None = ...) -> _FieldsetSpec: ...
     def get_inlines(self, request: HttpRequest, obj: _ModelT | None) -> list[type[InlineModelAdmin]]: ...
-    def get_ordering(self, request: HttpRequest) -> Sequence[str]: ...
-    def get_readonly_fields(self, request: HttpRequest, obj: _ModelT | None = ...) -> Sequence[str]: ...
+    def get_ordering(self, request: HttpRequest) -> _ListOrTuple[str]: ...
+    def get_readonly_fields(self, request: HttpRequest, obj: _ModelT | None = ...) -> _ListOrTuple[str]: ...
     def get_prepopulated_fields(self, request: HttpRequest, obj: _ModelT | None = ...) -> dict[str, Sequence[str]]: ...
     def get_queryset(self, request: HttpRequest) -> QuerySet[_ModelT]: ...
-    def get_sortable_by(self, request: HttpRequest) -> Sequence[str]: ...
+    def get_sortable_by(self, request: HttpRequest) -> _ListOrTuple[str] | _DisplayT: ...
```

Comment:
This was a little surprise if-statement found that returns `get_list_display` as a default:

https://github.com/django/django/blob/d254a54e7f65e83d8971bd817031bc6af32a7a46/django/contrib/admin/options.py#L424-L430

Not sure if we want to refine this to only be `_DisplayT` or something?

Reply:
Looks like any `_ListOrTuple[str]` is compatible with `_DisplayT`, so makes sense.

Reply:
I updated to only have `_DisplayT`